src/dmd/cli.d

```diff
@@ -763,6 +761,11 @@ dmd -cov -unittest myprog.d
             "disable access to shared memory objects"),
         Feature("in", "previewIn",
             "`in` on parameters means `scope const [ref]` and accepts rvalues"),
+        // DEPRECATED previews
+        Feature("dip25", "useDIP25",
+            "implement https://github.com/dlang/DIPs/blob/master/DIPs/archive/DIP25.md (Sealed references)", false, true),
+        // trigger deprecation message once D repositories don't use this flag anymore
+        Feature("markdown", "markdown", "enable Markdown replacements in Ddoc", false, false),
```

Comment:
Druntime and Phobos are compiled with `-de`, so this must be done as a multi-stage PR iteration.